File: Tl2.4 RESTFULL_DOTNET_EUREKA_BANK_GR08/01.SERVIDOR/ws_eureka_bank_restful_dotnet/ec.edu.monster.servicio/MovimientoService.cs
using ec.edu.monster.dao;
using ec.edu.monster.model;

namespace ec.edu.monster.servicio;

public class MovimientoService
{
    private readonly MovimientoDAO _dao;

    public MovimientoService(IConfiguration configuration)
    {
        _dao = new MovimientoDAO(configuration);
    }

    public List<MovimientoModel> ObtenerMovimiento(string cuenta)
    {
        return _dao.ObtenerMovimientosPorCuenta(cuenta);
    }
}

File: TI2.1  SOAP_JAVA_EUREKA_BANK_GR08/01.SERVIDOR/ws_eureka_bank_soap_java/src/main/java/ec/edu/monster/servicio/MovimientoService.java
package ec.edu.monster.servicio;

import ec.edu.monster.Controlador.MovimientoDAO;
import ec.edu.monster.modelo.MovimientoModel;
import java.sql.SQLException;
import java.util.List;

public class MovimientoService {

    MovimientoDAO DAO = new MovimientoDAO();

    public List<MovimientoModel> ObtenerMovimiento(String cuenta) throws SQLException {
        return DAO.obtenerMovimientosPorCuenta(cuenta);
    }
}
